plans.go

```diff
@@ -36,6 +36,9 @@ type Plan struct {
 	AvailableLocations       []ShortLocation             `json:"available_locations"`
 	AvailableOsImageVersions []ShortOsImageVersion       `json:"available_os_image_versions"`
 	AvailableApplications    []ShortOsImageVersion       `json:"available_applications"`
+	Netfilter                Netfilter                   `json:"netfilter"`
+	PPP                      PPP                         `json:"ppp"`
+	TunTap                   TUNTAP                      `json:"tun_tap"`
```

Comment:
Property should be `TUNTAP` as well. Here is a common convention in Go, all abbreviations should be capitalized.